backend/api/coworking/reservation.py

```diff
@@ -36,7 +36,10 @@ def draft_reservation(
     reservation_svc: ReservationService = Depends(),
 ) -> Reservation:
     """Draft a reservation request."""
-    return reservation_svc.draft_reservation(subject, reservation_request)
+    try:
+        return reservation_svc.draft_reservation(subject, reservation_request)
+    except ReservationException as e:
+        raise HTTPException(status_code=422, detail=(str(e)))
```

Comment:
If `ReservationException` is handled in the middleware, it would be fine letting this bubble up without custom error handling here.